### DISCUSSION TITLE
jsconfig.json is malformed JSON5

### DISCUSSION THREAD

**Discussion by savio-sandesh** [shadcn-ui/ui#6174] [General]
PS C:\Users\mrrea\OneDrive\Desktop\LMS_APP\client> npx shadcn@latest init
>>
✔ Preflight checks.
⠋ Verifying framework.
Something went wrong. Please check the error below for more details.   
If the problem persists, please open an issue on GitHub.

C:\Users\mrrea\OneDrive\Desktop\LMS_APP\client\jsconfig.json is malformed JSON5: invalid end of input at 1:1 

I am using JavaScript instead of TypeScript in my project but while setting up shadowcn my compiler throws an error after lots of try i am unable to fix it . Help me

**Comment by Dinobondhu-shill** [comment]
did you fix it later ? if yes please help me with the solution

**Reply by TruoYang** [reply]
I found this is caused by the latest shadcn-ui get some error, so may you can use `npx shadcn-ui@0.8.0 init ` to fix that. Or it just because the file jsconfig.json is empty, you need to add an `{}` into it. Well, I tried two ways but still cannot fix hmu T_T

**Reply by TruoYang** [reply]
I tried with `npx shadcn-ui@0.7.0 init` and it success =)) Don't understand much
